semcore/dropdown/src/Dropdown.jsx

```diff
@@ -100,20 +102,27 @@ class Dropdown extends Component {
   }
 
   getPopperProps() {
-    const { uid, disablePortal, ignorePortalsStacking } = this.asProps;
+    const { uid, disablePortal, ignorePortalsStacking, interaction } = this.asProps;
 
     return {
       id: `igc-${uid}-popper`,
       tabIndex: 0,
       disablePortal,
       ignorePortalsStacking,
       ref: this.popperRef,
+      focusMaster: interaction === 'click',
     };
   }
 
   render() {
     const { Children, forwardRef, modifiers = [], stretch, ...other } = this.asProps;
 
+    logger.warn(
+      other.interaction !== 'click' && other.interaction !== 'focus',
```

Comment:
Why not just `other.interaction !== 'hover'`

Reply:
maybe you mean `other.interaction === 'hover'` ?

But there is another options in interactions, and Dropdown should work only with `click` or `focus`